package.json

```diff
@@ -51,11 +54,17 @@
   },
   "devDependencies": {
     "@babel/core": "^7.12.10",
+    "@storybook/addon-actions": "^6.3.6",
+    "@storybook/addon-essentials": "^6.3.6",
+    "@storybook/addon-links": "^6.3.6",
+    "@storybook/react": "^6.3.6",
     "@testing-library/react-hooks": "^4.0.0",
     "babel-jest": "^26.6.3",
+    "babel-loader": "^8.2.2",
     "babel-plugin-module-resolver": "^4.1.0",
     "babel-plugin-relay": "^10.1.2",
     "babel-plugin-transform-dynamic-import": "^2.1.0",
+    "chromatic": "^5.10.0-canary.0",
```

Comment:
Any reason to use canary?

Reply:
no i just ran yarn chromatic.  Seems to be working just fine for now tho